### DISCUSSION TITLE
[V3] title() fluent method don't work when using #[Layout] on component

### DISCUSSION THREAD

**Discussion by joaokamun** [livewire/livewire#5930] [7. Bugs (archived)]
**Failing test:** https://github.com/livewire/livewire/pull/5961

I tried using the Layout + Title attributes on the full page components, and it's beautiful how it hides entirely the render() function.

I was also experimenting with dynamic titles. Following the documentation, I added a render method with the title fluent method, as result both layout and view was rendered correctly, but the title remained unfilled, after trying a few things I removed the Layout attribute and used it as a fluent method (like in V2) and it worked. 

**$title not being filled:**
```
#[Layout('components.layouts.app')]
class Example extends Component
{
    public Model $model;

    public function render()
    {
        return view('livewire.example')
            ->title($this->model->formatted_title);
    }
}
```

**Working:**
```
class Example extends Component
{
    public Model $model;

    public function render()
    {
        return view('livewire.example')
            ->layout('components.layouts.app')
            ->title($this->model->formatted_title);
    }
}
```

**Comment by joshhanley** [comment]
@joaokamun thanks for reporting! Hmm yeah I suspect it might not because it might need the `->layout()` to be passed so the `->title()` method get's called on it. But I agree it should ideally work. Any chance you could submit a failing test PR so we can investigate it? Thanks!

**Reply by joaokamun** [reply]
@joshhanley sure, just submitted failing test PR and edited my original message.

**Reply by joshhanley** [reply]
@joaokamun thanks!
